examples/validation-app/src/interceptors/validate-phone-num.interceptor.ts

```diff
@@ -60,7 +60,8 @@ export class ValidatePhoneNumInterceptor implements Provider<Interceptor> {
       const err: ValidationError = new ValidationError(
         'Area code and city do not match',
       );
-      err.statusCode = 422;
+      // change the error status code to 400 - "Bad Request"
+      err.statusCode = 400;
```

Comment:
Just wondering, why the update here?

Reply:
let me change it back. :)

Reply:
UPDATED: I recalled why I set it to `400`. Since I've handled the error coming from this interceptor, I want to set it to be 400 (bad request) rather than 422 (Unprocessable Entity) error. Otherwise, the error in here will be processed again in the `sequence.ts`.  Hope it makes sense.